Tasks/google-play-promote/task.json

```diff
@@ -14,7 +14,7 @@
     ],
     "version": {
         "Major": "1",
-        "Minor": "129",
+        "Minor": "130",
```

Comment:
Can you please update this to 135? We follow a convention of VSTS sprint numbers :).

Reply:
Updated all version numbers to 135 :)